net-im/corebird/corebird-1.5.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+VALA_MIN_API_VERSION=0.28
+
+AUTOTOOLS_AUTORECONF=1
+AUTOTOOLS_IN_SOURCE_BUILD=1
+inherit eutils autotools-utils gnome2 vala
```

Comment:
EAPI=6 doesn't even allow it anymore, so that's a no-brainer even.
Don't know why that was in the previous ebuilds - especially the in-source-build stuff.